tests/phpunit/tests/rest-api/wpRestTemplatesController.php

```diff
@@ -845,4 +845,46 @@ public function test_get_template_fallback() {
 		$response = rest_get_server()->dispatch( $request );
 		$this->assertSame( 'index', $response->get_data()['slug'], 'Should fallback to `index.html` when  ignore_empty is `true`.' );
 	}
+
+	/**
+	 * @ticket 57851
+	 *
+	 * @covers WP_REST_Templates_Controller::prepare_item_for_database
+	 */
+	public function test_prepare_item_for_database() {
+		$endpoint = new WP_REST_Templates_Controller( 'wp_template_part' );
+
+		$prepare_item_for_database = new ReflectionMethod( $endpoint, 'prepare_item_for_database' );
+		$prepare_item_for_database->setAccessible( true );
+
+		$body_params = array(
+			'title'   => 'Untitled Template Part',
+			'slug'    => 'untitled-template-part',
+			'content' => '',
+		);
+
+		$request = new WP_REST_Request( 'POST', '/wp/v2/template-parts' );
+		$request->set_body_params( $body_params );
+
+		$prepared = $prepare_item_for_database->invoke( $endpoint, $request );
+
+		$this->assertInstanceOf( 'stdClass', $prepared );
```

Comment:
Test methods with multiple assertions should have a final `$message` parameter at the end of each assertion to provide context should the assertion fail. See [Writing PHPUnit Tests - Using Assertions](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions) in the handbook.

For example:
```php
$this->assertInstanceOf( 'stdClass', $prepared, 'The item could not be prepared for the database.' );
```

Reply:
I updated to include the `$message` parameter. Thanks.